README.md

```diff
@@ -8,6 +8,17 @@ The application consist of two stateless services, one hosting an ASP.Net Core a
 
 It uses a structured logging framework called [Serilog](https://serilog.net/) to log events. The logged events are then written to [Application Insights](https://azure.microsoft.com/en-us/services/application-insights/) using [this](https://github.com/serilog/serilog-sinks-applicationinsights) Serilog sink. By using Serilog to capture the events instead of logging directly to Application Insights using the SDK it is easy to add different outputs for the logged events.
 
+Also this repository contains [NuGet package](https://www.nuget.org/packages/ServiceFabric.Logging/) that provides logging and tracing for Azure Service Fabric services.
```

Comment:
I would change it to "Also this repository contains the  [NuGet package](https://www.nuget.org/packages/ServiceFabric.Logging/) ServiceFabric.Logging"